docs/user/science.rst

```diff
@@ -0,0 +1,65 @@
+Read the Docs for Science
+=========================
+
+Documentation and technical writing are broad fields that include many science activities - building publications, academic writing, educational resources and interactive data science and data journalism.
+
+Let's explore the overlap between features needed for software documentation and for instance academic writing. Here's a brief overview of some of the features that people in science and academic writing love about the many tools used on Read the Docs:
+
+🪄 Easy to use
+  You don't have to be a programmer. Use familiar Markdown-derived syntax and write your formulas with LaTeX.
+🔣 Formulas and proofs
+  Use symbols familiar from math and physics, build advanced proofs.
+🧮 Results directly imported
+  Graphs, tables etc. are computed when the latest version of your website is built. Your results can be directly imported in their latest versions.
+📐 Widgets, widgets and more widgets
+  Design your science project's layout and components with widgets from a rich eco-system of Open Source extensions built for many purposes.
```

Comment:
What are widgets? Example here might help.

Reply:
Good point! This will be elaborated in the new PR